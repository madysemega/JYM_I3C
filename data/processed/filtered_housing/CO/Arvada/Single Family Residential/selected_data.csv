period_begin,median_sale_price,median_list_price,median_ppsf,median_list_ppsf,homes_sold,pending_sales,new_listings,inventory,months_of_supply,median_dom,avg_sale_to_list,sold_above_list,price_drops,off_market_in_two_weeks,remote_work_share,asset_value_momentum,State,City,Property_Type
2019-01-31,418250.0,425000.0,186.16365839427,186.90530131400087,80.0,130.0,131.0,210.0,2.6,33.0,0.9888368497622396,0.2124999999999999,0.3095238095238095,0.4461538461538461,0.41,0.0,CO,Arvada,Single Family Residential
2019-02-28,440000.0,440000.0,182.3176823176823,186.6720192693697,129.0,129.0,149.0,191.0,1.5,27.0,0.991471632681398,0.2403100775193798,0.3664921465968586,0.689922480620155,0.41,-0.020659113114560812,CO,Arvada,Single Family Residential
2019-03-31,466000.0,450000.0,177.35760105250944,183.03660982948847,174.0,157.0,209.0,214.0,1.2,14.0,0.9945404540737898,0.2816091954022988,0.4065420560747663,0.662420382165605,0.65,-0.02720570600787975,CO,Arvada,Single Family Residential
2019-04-30,476500.0,439750.0,182.35467689583948,185.3901855048933,169.0,193.0,232.0,237.0,1.4,10.0,0.9976040941185448,0.3609467455621301,0.4556962025316455,0.6113989637305699,0.59,0.02817514340335814,CO,Arvada,Single Family Residential
2019-05-31,466500.0,447000.0,177.85843920145192,182.5296369923219,223.0,231.0,283.0,277.0,1.2,10.0,0.9983466704015594,0.3228699551569506,0.4981949458483754,0.593073593073593,0.68,-0.02465655266388256,CO,Arvada,Single Family Residential
2019-06-30,472250.0,449050.0,178.92009177784848,190.74440120875937,202.0,226.0,249.0,284.0,1.4,11.0,0.9991324170914596,0.3118811881188118,0.4753521126760563,0.579646017699115,0.74,0.005969087444841925,CO,Arvada,Single Family Residential
2019-07-31,450000.0,429000.0,180.9805857189865,187.5,225.0,193.0,192.0,262.0,1.2,12.0,0.999041799824172,0.3422222222222222,0.5839694656488549,0.533678756476684,0.59,0.011516280372225518,CO,Arvada,Single Family Residential
2019-08-31,476000.0,420000.0,184.55309882316664,191.7053990610329,206.0,159.0,188.0,260.0,1.3,19.0,0.9959500902574216,0.2864077669902912,0.5076923076923076,0.4716981132075471,0.87,0.019739758770187965,CO,Arvada,Single Family Residential
2019-09-30,445000.0,440000.0,179.52901573218963,184.9112426035503,152.0,158.0,210.0,287.0,1.9,21.0,0.9893688699392156,0.2236842105263157,0.4634146341463414,0.5063291139240506,1.15,-0.027222967931797903,CO,Arvada,Single Family Residential
2019-10-31,450000.0,427750.0,182.1939307192989,190.29128213579435,176.0,161.0,149.0,248.0,1.4,19.0,0.9931480961237792,0.25,0.5685483870967742,0.4037267080745341,1.62,0.014843923564336947,CO,Arvada,Single Family Residential
2019-11-30,443750.0,441950.0,179.51528599605524,185.42431502994685,136.0,121.0,92.0,193.0,1.4,25.0,0.9893979849233242,0.2426470588235294,0.4404145077720207,0.3719008264462809,2.0,-0.014702162210719072,CO,Arvada,Single Family Residential
2019-12-31,450000.0,455350.0,183.30103359173123,197.9826963834668,145.0,92.0,75.0,153.0,1.1,33.0,0.9892657750428268,0.2482758620689655,0.2941176470588235,0.3478260869565217,2.83,0.0210887199642662,CO,Arvada,Single Family Residential
2020-01-31,485000.0,449000.0,173.98648648648648,198.5553549939832,97.0,119.0,151.0,158.0,1.6,33.0,0.9912059213070756,0.2371134020618556,0.2974683544303797,0.6974789915966386,3.23,-0.05081557328253339,CO,Arvada,Single Family Residential
2020-02-29,467950.0,469450.0,193.72458813432937,197.23710473789265,114.0,138.0,149.0,140.0,1.2,7.0,1.0099219615977328,0.5,0.2142857142857142,0.6956521739130433,3.71,0.11344617646138833,CO,Arvada,Single Family Residential
2020-03-31,515000.0,470000.0,192.5415484394001,196.90269473339336,183.0,163.0,195.0,161.0,0.9,8.0,1.0095218268360713,0.4699453551912568,0.2546583850931676,0.8159509202453987,3.54,-0.006106812286053032,CO,Arvada,Single Family Residential
2020-04-30,502500.0,450000.0,195.90653859384508,192.96340973059912,138.0,102.0,135.0,182.0,1.3,5.0,1.007630815517228,0.5,0.1538461538461538,0.6274509803921569,4.43,0.017476696233717393,CO,Arvada,Single Family Residential
2020-05-31,470000.0,467000.0,189.66235931638184,201.64986251145737,119.0,204.0,223.0,187.0,1.6,11.0,0.9975792008990544,0.2941176470588235,0.3850267379679144,0.6666666666666666,4.9,-0.03187325610611047,CO,Arvada,Single Family Residential
2020-06-30,499000.0,465000.0,184.9112426035503,207.26829268292684,197.0,224.0,196.0,139.0,0.6999999999999998,9.0,1.00185919712786,0.4263959390862944,0.5539568345323741,0.6964285714285714,5.51,-0.02505039339358872,CO,Arvada,Single Family Residential
2020-07-31,505500.0,465000.0,196.8484072075861,205.0832605043965,244.0,204.0,219.0,134.0,0.5,6.0,1.0088081995982283,0.4672131147540983,0.4701492537313433,0.7401960784313725,5.47,0.06455618617862546,CO,Arvada,Single Family Residential
2020-08-31,500000.0,456000.0,195.96864501679727,211.92528735632183,211.0,205.0,205.0,122.0,0.5999999999999999,6.0,1.0093173038489232,0.4786729857819905,0.4098360655737705,0.7414634146341463,5.67,-0.004469237030000794,CO,Arvada,Single Family Residential
2020-09-30,523000.0,483500.0,204.2735461947217,204.75034208902363,204.0,205.0,199.0,106.0,0.5,5.0,1.011786230927514,0.5049019607843137,0.5660377358490565,0.7365853658536585,5.79,0.04237872429649436,CO,Arvada,Single Family Residential
2020-10-31,528497.5,478000.0,203.1044034627141,209.04997352665663,224.0,181.0,172.0,92.0,0.4,5.0,1.0161573982246712,0.5446428571428571,0.3804347826086956,0.7679558011049724,6.54,-0.005723417220618088,CO,Arvada,Single Family Residential
2020-11-30,510000.0,490000.0,203.7037037037037,212.13073538654933,169.0,149.0,122.0,58.0,0.2999999999999999,5.0,1.0159041150744073,0.5798816568047336,0.396551724137931,0.7449664429530201,6.1,0.002950700382523319,CO,Arvada,Single Family Residential
2020-12-31,535000.0,475000.0,206.55965807911167,223.4848484848485,158.0,79.0,83.0,47.0,0.2999999999999999,5.0,1.0150395217705863,0.5569620253164556,0.3404255319148936,0.7721518987341772,6.29,0.014020139661093722,CO,Arvada,Single Family Residential
2021-01-31,540000.0,495000.0,216.51008300357853,229.6973493726867,80.0,127.0,125.0,40.0,0.5,4.0,1.0193474207874509,0.5749999999999998,0.325,0.84251968503937,5.45,0.04817216012555492,CO,Arvada,Single Family Residential
2021-02-28,555000.0,520000.0,216.299729625338,224.78230980751604,122.0,144.0,154.0,49.0,0.4,4.0,1.0381755962918358,0.721311475409836,0.2448979591836734,0.9027777777777776,6.69,-0.000971563888953253,CO,Arvada,Single Family Residential
2021-03-31,579000.0,499500.0,227.69238894283228,231.74689194347968,167.0,164.0,161.0,43.0,0.2999999999999999,4.0,1.0583834230547096,0.7664670658682635,0.3255813953488372,0.9207317073170732,6.6,0.05267070530891549,CO,Arvada,Single Family Residential
2021-04-30,575000.0,549950.0,242.2427871529668,242.61138067931188,168.0,185.0,229.0,83.0,0.5,4.0,1.0540029740543642,0.7380952380952381,0.144578313253012,0.9405405405405404,7.09,0.0639037531192479,CO,Arvada,Single Family Residential
2021-05-31,600000.0,518950.0,230.36419482229047,240.89338475081885,189.0,198.0,185.0,68.0,0.4,5.0,1.0687163841079357,0.8148148148148147,0.2352941176470588,0.8888888888888888,6.37,-0.04903589687966836,CO,Arvada,Single Family Residential
2021-06-30,602250.0,560000.0,253.4116301340711,236.9573874950219,196.0,228.0,255.0,94.0,0.5,5.0,1.06921497435448,0.8163265306122449,0.4468085106382978,0.881578947368421,5.57,0.10004781919151995,CO,Arvada,Single Family Residential
2021-07-31,625250.0,550000.0,226.57641610260063,241.923774954628,215.0,203.0,246.0,134.0,0.5999999999999999,5.0,1.03914481075279,0.6790697674418604,0.4253731343283582,0.7733990147783251,5.21,-0.10589574763112841,CO,Arvada,Single Family Residential
2021-08-31,580000.0,549000.0,240.5492418497585,256.59824046920824,205.0,217.0,211.0,127.0,0.5999999999999999,6.0,1.029218013673216,0.697560975609756,0.5118110236220472,0.7235023041474654,4.42,0.06166937401300654,CO,Arvada,Single Family Residential
2021-09-30,580500.0,565000.0,242.94294034603752,244.86226497595104,196.0,186.0,220.0,161.0,0.8,5.0,1.0237463354112943,0.6122448979591836,0.4782608695652174,0.7365591397849462,4.73,0.009950970860985286,CO,Arvada,Single Family Residential
2021-10-31,610000.0,568850.0,245.77662790393936,258.70646766169153,198.0,183.0,167.0,139.0,0.6999999999999998,9.0,1.0181766118247206,0.5606060606060606,0.4460431654676259,0.6612021857923496,4.53,0.011664004534833072,CO,Arvada,Single Family Residential
2021-11-30,615000.0,550000.0,239.5110670630988,265.0093808630394,179.0,161.0,127.0,92.0,0.5,6.0,1.016933472227984,0.5865921787709497,0.358695652173913,0.7080745341614906,4.99,-0.025492907500095585,CO,Arvada,Single Family Residential
2021-12-31,575000.0,551500.0,247.23874256584537,268.82907221806647,156.0,92.0,67.0,56.0,0.4,6.0,1.0207483535738455,0.532051282051282,0.4285714285714285,0.6521739130434782,5.36,0.03226437758181211,CO,Arvada,Single Family Residential
2022-01-31,622500.0,587000.0,247.79593830847983,276.24309392265195,100.0,102.0,91.0,39.0,0.4,13.0,1.01614411627058,0.4699999999999999,0.2051282051282051,0.803921568627451,5.49,0.0022536748765662473,CO,Arvada,Single Family Residential
2022-02-28,680500.0,625000.0,265.1442602823818,265.092065723853,105.0,153.0,169.0,51.0,0.5,4.0,1.0735425573867872,0.780952380952381,0.196078431372549,0.934640522875817,5.21,0.07001051789761448,CO,Arvada,Single Family Residential
2022-03-31,700000.0,625000.0,278.8844621513944,270.5882352941176,168.0,165.0,197.0,80.0,0.5,4.0,1.0906790478310402,0.8630952380952381,0.3125,0.9333333333333332,4.9,0.05182160780844036,CO,Arvada,Single Family Residential
2022-04-30,700000.0,615000.0,277.8354444698677,274.7784991868646,177.0,184.0,218.0,105.0,0.5999999999999999,5.0,1.080548963263674,0.8248587570621468,0.3523809523809524,0.8967391304347825,5.01,-0.003761477686617276,CO,Arvada,Single Family Residential
2022-05-31,658500.0,614975.0,276.50560322404647,278.5406041584935,198.0,212.0,238.0,129.0,0.6999999999999998,5.0,1.0519509603511448,0.6717171717171717,0.4961240310077519,0.839622641509434,5.26,-0.004786434820649554,CO,Arvada,Single Family Residential
2022-06-30,649950.0,620325.0,276.6021218977188,272.7223733159103,172.0,170.0,246.0,201.0,1.2,6.0,1.025496665717828,0.6104651162790697,0.626865671641791,0.6529411764705882,5.28,0.00034906588708127906,CO,Arvada,Single Family Residential
2022-07-31,653250.0,600000.0,258.0817569318476,272.6347040714669,141.0,175.0,204.0,227.0,1.6,12.0,1.0026001102815307,0.375886524822695,0.6696035242290748,0.56,5.31,-0.06695669880912769,CO,Arvada,Single Family Residential
2022-08-31,340000.0,587500.0,142.3327937107052,270.5061082024433,74.5,151.0,149.0,214.0,1.4,70.0,0.8461498904529599,0.1858108108108108,0.6542056074766354,0.390728476821192,5.76,-0.4484972692266218,CO,Arvada,Single Family Residential
2022-09-30,642500.0,600000.0,248.93730074388947,271.6004374772147,140.0,126.0,170.0,240.0,1.7,15.0,0.9875158343106016,0.2928571428571428,0.5916666666666666,0.4761904761904761,6.04,0.748980640750019,CO,Arvada,Single Family Residential
2022-10-31,625000.0,597000.0,244.3358507330076,260.280549858589,104.0,105.0,113.0,233.0,2.2,17.0,0.9892363966784258,0.298076923076923,0.6523605150214592,0.3333333333333333,6.1,-0.01848437336281683,CO,Arvada,Single Family Residential
2022-11-30,595000.0,562475.0,248.40426724448736,265.4982466569392,92.0,100.0,83.0,186.0,2.0,27.0,0.9818081245874636,0.1304347826086956,0.5967741935483871,0.33,5.26,0.01665091921334727,CO,Arvada,Single Family Residential
2022-12-31,590000.0,622000.0,241.6998671978752,260.4940131678408,86.0,76.0,48.0,134.0,1.6,33.0,0.9813245947386124,0.1395348837209302,0.4328358208955223,0.3684210526315789,4.79,-0.02698987469492009,CO,Arvada,Single Family Residential
2023-01-31,610000.0,599975.0,251.43090760425184,274.1816423753596,78.0,89.0,84.0,91.0,1.2,42.0,0.9784142480146704,0.1666666666666666,0.4395604395604395,0.5168539325842697,4.51,0.04026084300000887,CO,Arvada,Single Family Residential
2023-02-28,590000.0,639000.0,258.8519872910794,268.4331797235023,120.0,99.0,119.0,97.0,0.8,41.0,0.984465800449334,0.3083333333333333,0.3092783505154639,0.7171717171717171,4.61,0.02951538359996775,CO,Arvada,Single Family Residential
2023-03-31,610000.0,630475.0,253.92159698420272,261.9347160292587,132.0,125.0,168.0,125.0,0.9,7.0,1.0116240888706818,0.4318181818181818,0.3519999999999999,0.7119999999999999,4.34,-0.019047141026321213,CO,Arvada,Single Family Residential
2023-04-30,688000.0,615000.0,262.19881771924577,258.0061755624173,122.0,128.0,154.0,145.0,1.2,5.0,1.0102081895167467,0.4672131147540983,0.3448275862068965,0.7265625,3.77,0.0325975452003715,CO,Arvada,Single Family Residential
2023-05-31,665000.0,629975.0,254.015301530153,271.25738223088933,150.0,150.0,141.0,128.0,0.9,7.0,1.0111719601660782,0.4666666666666666,0.640625,0.6066666666666666,3.96,-0.03121111018073086,CO,Arvada,Single Family Residential
2023-06-30,688000.0,619500.0,258.36280680256954,270.21649867625933,142.0,130.0,153.0,139.0,1.0,7.0,1.0093858639834543,0.4436619718309859,0.4028776978417266,0.6307692307692307,4.07,0.017115131435892827,CO,Arvada,Single Family Residential
2023-07-31,644500.0,600000.0,267.09366547787164,269.6673189823875,121.0,117.0,132.0,136.0,1.1,11.0,1.0035768980997333,0.3801652892561983,0.4852941176470588,0.5982905982905982,4.07,0.03379301681752467,CO,Arvada,Single Family Residential
2023-08-31,660000.0,599947.5,261.1287758346582,277.0902470551823,120.0,95.0,146.0,181.0,1.5,14.0,0.9958752290061866,0.3833333333333333,0.5303867403314916,0.5684210526315789,3.8,-0.022332576223930034,CO,Arvada,Single Family Residential
2023-09-30,677236.0,599000.0,255.67590987868283,277.26109215017067,90.0,126.0,152.0,194.0,2.2,9.0,0.9907442933577418,0.3888888888888889,0.5876288659793814,0.5238095238095238,3.14,-0.02088190372181742,CO,Arvada,Single Family Residential
