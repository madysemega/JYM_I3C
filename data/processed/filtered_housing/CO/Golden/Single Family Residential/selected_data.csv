period_begin,median_sale_price,median_list_price,median_ppsf,median_list_ppsf,homes_sold,pending_sales,new_listings,inventory,months_of_supply,median_dom,avg_sale_to_list,sold_above_list,price_drops,off_market_in_two_weeks,remote_work_share,asset_value_momentum,State,City,Property_Type
2019-01-31,592000.0,725000.0,227.42988859008835,226.6454352441613,19.0,20.0,18.0,28.0,1.5,26.0,0.9888794682060564,0.263157894736842,0.2857142857142857,0.5,0.47,0.0,CO,Golden,Single Family Residential
2019-02-28,595000.0,644900.0,228.94290559638213,231.81589114732068,23.0,14.0,25.0,34.0,1.5,31.0,0.9922595786098872,0.3043478260869565,0.1764705882352941,0.5714285714285714,1.57,0.006652674438146544,CO,Golden,Single Family Residential
2019-03-31,404250.0,381475.0,130.76039579167224,145.75267314815642,9.0,25.0,19.5,22.5,3.2,101.5,0.9432528675000138,0.1470588235294117,0.3902439024390244,0.5999999999999999,2.52,-0.42885150578896747,CO,Golden,Single Family Residential
2019-04-30,627500.0,602700.0,227.1301855673,232.08973021230216,26.0,38.0,44.0,47.0,1.8,8.0,0.9930989748424568,0.3076923076923077,0.4468085106382978,0.6842105263157894,2.79,0.7369952437981631,CO,Golden,Single Family Residential
2019-05-31,367900.0,363675.0,129.52359198742172,149.19214094894988,20.0,17.5,24.0,30.0,2.7,35.5,0.9845553871694964,0.1794871794871794,0.375,0.3529411764705882,2.61,-0.42973853667264705,CO,Golden,Single Family Residential
2019-06-30,369833.25,585000.0,137.55230775559508,245.606128886886,19.0,16.0,45.0,34.0,1.4,87.5,0.9661948350191204,0.2222222222222222,0.4393939393939393,0.3225806451612903,2.28,0.061986512611177824,CO,Golden,Single Family Residential
2019-07-31,663750.0,645000.0,221.71542332002664,238.9540126239856,32.0,37.0,38.0,63.0,2.0,10.0,1.0004935516085107,0.375,0.5079365079365079,0.5135135135135134,3.19,0.6118626211199145,CO,Golden,Single Family Residential
2019-08-31,359129.25,384100.0,140.8842852431294,149.84549743490027,23.5,30.0,17.0,29.0,1.6,63.0,0.996138804606568,0.1413043478260869,0.5357142857142857,0.3666666666666666,3.69,-0.3645715614480488,CO,Golden,Single Family Residential
2019-09-30,673500.0,650000.0,210.49592119269715,230.94819761695987,20.0,28.0,30.0,51.0,2.6,52.0,0.9802698174797566,0.1499999999999999,0.4117647058823529,0.3571428571428571,4.13,0.4941050439332981,CO,Golden,Single Family Residential
2019-10-31,350125.0,687500.0,127.62334948701825,254.4494596960588,22.5,25.0,19.0,18.0,1.15,216.5,0.981713927242066,0.3313953488372093,0.5454545454545454,0.3599999999999999,3.2,-0.3937015560021885,CO,Golden,Single Family Residential
2019-11-30,401125.0,342700.0,157.255015388608,159.42909653229916,9.5,8.5,8.0,15.0,1.3,46.0,0.9631510072042748,0.0588235294117647,0.2142857142857142,0.2333333333333333,2.54,0.2321806003422897,CO,Golden,Single Family Residential
2019-12-31,620000.0,642500.0,230.4201820603908,269.7416883150526,20.0,13.0,10.0,23.0,1.2,21.0,0.982304206792911,0.2,0.2608695652173913,0.5384615384615384,2.08,0.4652644399987964,CO,Golden,Single Family Residential
2020-01-31,745000.0,761500.0,207.9606440071556,295.35729371361407,13.0,9.0,7.0,15.0,1.2,22.0,0.9823822838860834,0.3076923076923077,0.4666666666666666,0.3333333333333333,2.29,-0.09747209577045113,CO,Golden,Single Family Residential
2020-02-29,749900.0,610000.0,236.63810689514483,234.4485594611985,13.0,9.0,13.0,16.0,1.2,59.0,0.9750491439929776,0.2307692307692307,0.375,0.7777777777777777,2.61,0.13789850971514817,CO,Golden,Single Family Residential
2020-03-31,715000.0,715000.0,227.2342995169082,240.75500770416025,13.0,23.0,40.0,32.0,2.5,4.0,0.9979264234177012,0.4615384615384615,0.1875,0.9130434782608696,2.77,-0.039739192903548215,CO,Golden,Single Family Residential
2020-04-30,410250.0,416950.0,138.64724150529776,159.9714285714286,11.0,16.0,10.0,18.0,2.3,121.5,0.963782785359448,0.2619047619047619,0.1818181818181818,0.4375,3.34,-0.3898489717438929,CO,Golden,Single Family Residential
2020-05-31,565100.0,785000.0,242.97024297024296,255.9808612440192,13.0,35.0,42.0,38.0,2.9,16.0,1.0002054231107604,0.3076923076923077,0.3947368421052631,0.6857142857142857,3.7,0.7524347425329698,CO,Golden,Single Family Residential
2020-06-30,449750.0,573000.0,152.53057436860726,260.95238095238096,21.5,20.5,45.0,21.5,2.45,38.5,0.965346738505692,0.2142857142857142,0.3846153846153846,0.3499999999999999,4.82,-0.3722252877390917,CO,Golden,Single Family Residential
2020-07-31,400000.0,656000.0,151.71656049112522,264.161220043573,21.5,22.5,38.0,15.5,0.8499999999999999,40.0,0.9786514787926316,0.1951219512195122,0.4482758620689655,0.2906976744186046,5.58,-0.005336725970197187,CO,Golden,Single Family Residential
2020-08-31,673237.5,689000.0,252.7565434989291,262.7174970655292,42.0,34.0,42.0,34.0,0.8,5.0,1.012043295321254,0.4285714285714285,0.2941176470588235,0.7941176470588234,6.48,0.6659786029997317,CO,Golden,Single Family Residential
2020-09-30,667000.0,759000.0,253.2326443542122,251.41420490257696,30.0,41.0,37.0,26.0,0.9,5.0,1.0133440611192135,0.6333333333333333,0.3461538461538461,0.7560975609756097,6.92,0.0018836341433239312,CO,Golden,Single Family Residential
2020-10-31,460000.0,700000.0,158.7023308805451,279.8191068400226,23.0,27.0,27.0,13.0,0.7999999999999999,207.0,0.978306149646304,0.2666666666666666,0.4799999999999999,0.6296296296296297,7.35,-0.3732943424997043,CO,Golden,Single Family Residential
2020-11-30,832500.0,720000.0,250.63637821903507,271.41512394101034,30.0,28.0,25.0,20.0,0.6999999999999998,15.0,1.0156783883566751,0.4666666666666666,0.65,0.5714285714285714,7.29,0.5792860560295647,CO,Golden,Single Family Residential
2020-12-31,692500.0,639000.0,253.85312783318224,232.9691516709512,29.0,16.0,12.0,12.0,0.4,6.0,1.021551101144144,0.586206896551724,0.3333333333333333,0.5,6.9,0.012834328508114723,CO,Golden,Single Family Residential
2021-01-31,600000.0,644950.0,241.1360185764044,292.4291938997821,13.0,19.0,19.0,11.0,0.8,13.0,1.0074750505486985,0.5384615384615384,0.2727272727272727,0.8421052631578946,6.61,-0.050096326822235526,CO,Golden,Single Family Residential
2021-02-28,421000.0,698700.0,152.2644927536232,280.5154431317843,13.0,12.0,24.0,6.0,0.7,36.0,1.0008802439096265,0.28,0.1818181818181818,0.4565217391304348,7.65,-0.3685535091250671,CO,Golden,Single Family Residential
2021-03-31,816000.0,705000.0,287.30936819172115,303.26548535384586,26.0,22.0,21.0,8.0,0.2999999999999999,5.0,1.037069407404664,0.6923076923076922,0.125,0.9545454545454544,8.46,0.8869098303608576,CO,Golden,Single Family Residential
2021-04-30,497500.0,720000.0,165.89096480166367,298.02631578947364,11.0,13.5,36.0,17.0,0.8,37.5,1.05053728083593,0.8095238095238095,0.1176470588235294,0.4423076923076922,9.19,-0.42260509691781145,CO,Golden,Single Family Residential
2021-05-31,880500.0,770000.0,327.31576603730593,297.89184234647115,29.0,49.0,51.0,18.0,0.5999999999999999,5.0,1.0955446669821454,0.7586206896551723,0.1666666666666666,0.8775510204081632,9.17,0.973077716611263,CO,Golden,Single Family Residential
2021-06-30,864500.0,707500.0,313.5663849326364,295.1766309468984,46.0,45.0,48.0,21.0,0.5,5.0,1.084537531568756,0.8043478260869565,0.3333333333333333,0.8888888888888888,9.22,-0.04200647366036869,CO,Golden,Single Family Residential
2021-07-31,846900.0,780000.0,312.9304407713498,318.05331179321485,35.0,42.0,59.0,38.0,1.1,6.0,1.0672219839587231,0.6571428571428571,0.4210526315789473,0.8095238095238095,8.52,-0.0020281005612997616,CO,Golden,Single Family Residential
2021-08-31,902500.0,699000.0,309.1436704846918,318.93520843797086,48.0,48.0,37.0,26.0,0.5,5.0,1.0433424404496363,0.5833333333333333,0.8846153846153845,0.5833333333333333,7.63,-0.012100996877529413,CO,Golden,Single Family Residential
2021-09-30,463750.0,492450.0,192.91917886879128,205.54614491199857,21.0,17.5,17.0,12.0,0.7999999999999999,31.0,0.973596122642824,0.2195121951219512,0.217391304347826,0.338235294117647,7.59,-0.37595623883768226,CO,Golden,Single Family Residential
2021-10-31,712000.0,839500.0,320.5128205128205,332.65988048819526,33.0,25.0,26.0,23.0,0.6999999999999998,6.0,1.03712057019061,0.606060606060606,0.0869565217391304,0.68,7.77,0.6613839141976059,CO,Golden,Single Family Residential
2021-11-30,476500.0,859000.0,203.6273093465233,328.9299658099389,15.5,12.5,13.0,5.5,0.6499999999999999,21.5,1.0378014966640126,0.8333333333333333,0.5,0.3541666666666666,8.01,-0.36468279483884725,CO,Golden,Single Family Residential
2021-12-31,492915.0,885000.0,192.47606770566443,383.33333333333326,10.5,5.5,10.0,7.0,0.4,38.0,0.9980872170079342,0.325,0.1428571428571428,0.3499999999999999,8.51,-0.054762996558001986,CO,Golden,Single Family Residential
2022-01-31,940000.0,760000.0,346.45313653136526,302.98369732390034,13.0,17.0,16.0,5.0,0.4,5.0,1.0593613848204029,0.7692307692307692,0.2,0.9411764705882352,8.49,0.7999803334571625,CO,Golden,Single Family Residential
2022-02-28,985500.0,855000.0,326.87392748913794,340.0,18.0,19.0,22.0,8.0,0.4,4.0,1.1048510486863738,0.7777777777777777,,0.894736842105263,7.94,-0.05651329711790554,CO,Golden,Single Family Residential
2022-03-31,1012250.0,799000.0,385.53398589058605,343.75,23.0,23.0,32.0,17.0,0.6999999999999998,5.0,1.0774891662432633,0.6956521739130433,0.2941176470588235,0.9565217391304348,6.84,0.1794577464530187,CO,Golden,Single Family Residential
2022-04-30,512500.0,453725.0,232.84948873908613,224.2536628880324,11.0,15.5,19.5,13.5,2.05,25.0,1.03203511819727,0.7857142857142857,0.125,0.4666666666666666,7.55,-0.39603381994663256,CO,Golden,Single Family Residential
2022-05-31,500000.0,462250.0,225.515211480296,244.18729831676265,20.0,31.0,30.5,12.5,1.7999999999999998,46.0,1.0363915055685289,0.358974358974359,0.4545454545454545,0.4180327868852459,8.5,-0.03149793155444025,CO,Golden,Single Family Residential
2022-06-30,486000.0,455950.0,251.63834951456312,211.8759081662308,26.5,17.0,28.0,23.0,0.95,25.0,1.0200435827177663,0.5147058823529411,0.3863636363636363,0.34375,9.68,0.11583758746380446,CO,Golden,Single Family Residential
2022-07-31,500000.0,463750.0,197.6420051346444,236.7760116137524,14.0,14.5,20.0,28.0,1.3,41.0,1.0038694591427446,0.3846153846153846,0.4909090909090909,0.2678571428571428,9.84,-0.21457915490259483,CO,Golden,Single Family Residential
2022-08-31,466250.0,428974.5,186.28628939943843,210.1012239815632,13.5,16.5,10.5,21.0,1.05,17.5,0.9739925581148084,0.14,0.6097560975609756,0.2258064516129032,9.42,-0.057455983243389186,CO,Golden,Single Family Residential
2022-09-30,401250.0,847000.0,225.0371633803969,336.24982393511345,15.0,12.0,36.0,51.0,1.8,31.0,0.9666490808963188,0.1034482758620689,0.5294117647058822,0.2826086956521738,9.14,0.2080178530899186,CO,Golden,Single Family Residential
2022-10-31,841500.0,629000.0,328.0063676955931,397.4293928631829,24.0,20.0,26.0,48.0,2.0,9.0,1.0055909849698044,0.4583333333333333,0.5416666666666666,0.55,10.4,0.4575653317365176,CO,Golden,Single Family Residential
2022-11-30,794950.0,687500.0,308.39465713292617,362.70161290322574,17.0,17.0,10.0,37.0,2.2,12.0,0.9813054842014016,0.2941176470588235,0.4054054054054054,0.2941176470588235,10.64,-0.05979063973802978,CO,Golden,Single Family Residential
2022-12-31,477500.0,557500.0,206.17875056999543,384.71295783893487,7.5,3.5,2.0,29.0,2.1,45.0,0.9876279196794868,0.5357142857142857,0.3103448275862069,0.1666666666666666,10.78,-0.3314451278540569,CO,Golden,Single Family Residential
2023-01-31,794500.0,732500.0,324.34229997608264,479.70442804428046,6.0,8.0,11.0,28.0,4.7,41.0,0.9708952538044456,0.1666666666666666,0.2857142857142857,0.75,9.28,0.5731121615560086,CO,Golden,Single Family Residential
2023-02-28,790000.0,765000.0,355.23978685612786,390.38101186758274,13.0,16.0,17.0,25.0,1.9,4.0,0.9935329079212986,0.3076923076923077,0.32,0.75,10.25,0.09526197132573722,CO,Golden,Single Family Residential
2023-03-31,817500.0,1000000.0,350.58459668704654,338.2318637742011,19.0,20.0,34.0,31.0,1.6,5.0,1.0230785339766817,0.4736842105263157,0.4838709677419355,0.8,10.36,-0.01310436032596396,CO,Golden,Single Family Residential
2023-04-30,532500.0,488187.5,210.56416823135623,206.2235171153505,11.5,12.5,13.5,16.0,2.15,21.0,0.9145806024601908,0.2272727272727272,0.3448275862068965,0.3541666666666666,9.96,-0.39939127325859436,CO,Golden,Single Family Residential
2023-05-31,536250.0,576225.0,178.32019044785002,235.08031714804133,13.0,29.0,19.0,18.5,2.65,60.5,1.0023072652202036,0.2,0.4848484848484848,0.8620689655172413,9.52,-0.15313136159082064,CO,Golden,Single Family Residential
2023-06-30,540083.25,950000.0,220.28436834274893,381.2425069246352,14.5,12.0,28.0,18.5,1.15,23.0,1.0033220628042554,0.4907407407407407,0.4642857142857142,0.2272727272727272,10.17,0.2353304905603013,CO,Golden,Single Family Residential
2023-07-31,937875.0,849950.0,339.59110125147845,354.49604086963126,24.0,23.0,35.0,42.0,1.8,23.0,0.9805192483995409,0.2916666666666666,0.5,0.6521739130434782,10.41,0.541603264027775,CO,Golden,Single Family Residential
2023-08-31,444000.0,413900.0,186.60406646320425,219.16458249709245,12.5,13.5,11.5,19.5,2.25,92.5,0.8136165083369435,0.1666666666666666,0.611111111111111,0.2115384615384615,10.33,-0.4505036622705323,CO,Golden,Single Family Residential
2023-09-30,511250.0,452000.0,230.0399544716865,213.3116883116884,11.5,24.0,14.5,20.5,2.85,30.0,0.9537899962029854,0.1136363636363636,0.4324324324324324,0.625,9.27,0.23277031863101016,CO,Golden,Single Family Residential
